db/migrate/20210830114756_remove_profile_picture_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveProfilePictureFromUsers < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :users, :profile_picture, :string
+  end
```

Comment:
So we're taking the profile picture out of this table in order to store it with activestorage